server/auth.js

```diff
@@ -42,6 +42,19 @@ internals.applyStrategy = function (server, next) {
                     }
 
                     done(null, Object.keys(results.user.roles));
+                }],
+                updateSession: ['scope', function (results, done) {
+
+                    if (!results.roles) {
+                        return done();
+                    }
+
+                    const update = {
+                        $set: {
+                            lastActive: new Date()
+                        }
+                    };
```

Comment:
add new line

Reply:
This has been added but does not close the comment